.github/workflows/format.yaml

```diff
@@ -38,10 +38,19 @@ jobs:
         if: contains(toJSON(steps.file_changes.outputs.files), 'bin/')
         id: check_bin_script_docs
         run: bin/help
-      
+
       - name: setup terraform
         uses: hashicorp/setup-terraform@v1
-      
+
       - name: terraform fmt
         id: tf_fmt
         run: terraform fmt -check -recursive -diff cloud
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@master
+        env:
+          SHELLCHECK_OPTS: -s bash -S error
+          SHFMT_OPTS:  -bn -ci -kp -i 2 # Google Bash Style
```

Comment:
Non-Blocking: For future readers, would you mind adding a link that explains each option. Or if there was a recipe that showed the options for Google Bash Style, link it instead?